torch/_inductor/codegen/cpp.py

```diff
@@ -151,6 +160,135 @@ def _print_IndexingDiv(self, expr):
 cexpr = CppPrinter().doprint
 
 
+class CppVecOverrides(OpOverrides):
+    """Map element-wise ops to aten vectorization C++"""
+
+    @staticmethod
+    def add(a, b):
+        return f"{a} + {b}"
```

Comment:
Isn't this (and many of these) the same as the default implementation in the base class?

Reply:
I added this to this class for vectorization support check. If the method is not placed in the `CppVecOverrides`, it is hard to decide whether an operation could be vectorized or not.